runtime/Java/src/org/antlr/v4/runtime/misc/IntervalSet.java

```diff
@@ -478,28 +478,43 @@ public String toString(boolean elemAreChar) {
 			return "{}";
 		}
 		if ( this.size()>1 ) {
-			buf.append("{");
+			buf.append('{');
 		}
 		Iterator<Interval> iter = this.intervals.iterator();
 		while (iter.hasNext()) {
 			Interval I = iter.next();
 			int a = I.a;
 			int b = I.b;
 			if ( a==b ) {
-				if ( a==Token.EOF ) buf.append("<EOF>");
-				else if ( elemAreChar ) buf.append("'").appendCodePoint(a).append("'");
-				else buf.append(a);
+				if ( a==Token.EOF ) {
```

Comment:
Seems like a lot of cosmetic changes here and flipping of strings two characters for no real game in performance. Yet it costs me mental effort to review

Reply:
It's not only cosmetic changes but clarity changes as well. `toString` should print clear representation of object instead of plain strings. In the old version it prints

`'
'` 

In the new version it prints `'\n'`.